nuxeo-mqueues-core/src/main/java/org/nuxeo/lib/core/mqueues/mqueues/kafka/KafkaUtils.java

```diff
@@ -18,56 +18,65 @@
  */
 package org.nuxeo.lib.core.mqueues.mqueues.kafka;
 
-import kafka.admin.AdminClient;
-import kafka.admin.AdminUtils;
-import kafka.admin.RackAwareMode;
-import kafka.cluster.Broker;
-import kafka.cluster.EndPoint;
-import kafka.coordinator.group.GroupOverview;
-import kafka.utils.ZKStringSerializer$;
-import kafka.utils.ZkUtils;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+
 import org.I0Itec.zkclient.ZkClient;
 import org.I0Itec.zkclient.ZkConnection;
 import org.I0Itec.zkclient.exception.ZkTimeoutException;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.apache.kafka.clients.admin.DescribeTopicsResult;
+import org.apache.kafka.clients.admin.NewTopic;
 import org.apache.kafka.clients.consumer.RangeAssignor;
 import org.apache.kafka.clients.consumer.RoundRobinAssignor;
 import org.apache.kafka.clients.consumer.internals.PartitionAssignor;
 import org.apache.kafka.common.Cluster;
 import org.apache.kafka.common.PartitionInfo;
 import org.apache.kafka.common.TopicPartition;
-import org.apache.kafka.common.requests.MetadataResponse;
+
+import kafka.admin.AdminClient;
+
+import kafka.admin.AdminUtils;
+import kafka.cluster.Broker;
+import kafka.cluster.EndPoint;
+import kafka.coordinator.group.GroupOverview;
+import kafka.utils.ZKStringSerializer$;
+import kafka.utils.ZkUtils;
 import org.nuxeo.lib.core.mqueues.mqueues.MQPartition;
+import scala.collection.Iterable;
+import scala.collection.IterableLike;
 import scala.collection.Iterator;
 import scala.collection.JavaConversions;
 import scala.collection.Seq;
 
-import java.time.Duration;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
-import java.util.Set;
-import java.util.stream.Collectors;
-
-
 /**
  * Misc Kafka Utils
- * @since 9.2
+ *
+ * @since 9.3
  */
 public class KafkaUtils implements AutoCloseable {
-    private static final Log log = LogFactory.getLog(KafkaUtils.class);
-    protected final ZkClient zkClient;
-    protected final ZkUtils zkUtils;
     public static final String DEFAULT_ZK_SERVER = "localhost:2181";
+
     public static final int ZK_TIMEOUT_MS = 6000;
+
     public static final int ZK_CONNECTION_TIMEOUT_MS = 10000;
 
+    private static final Log log = LogFactory.getLog(KafkaUtils.class);
```

Comment:
We tend to put `private static final Log log` as the first field (or second after `serialVersionUID` when present).